recipes/install_ext_services.rb

```diff
@@ -22,8 +22,8 @@
 
 case node['abiquo']['profile']
 when "monolithic", "server"
-    packages = %w{MariaDB-server MariaDB-client redis rabbitmq-server}
-    services = %w{mysql redis rabbitmq-server}
+    packages = %w{MariaDB-server MariaDB-client redis rabbitmq-server cronie}
```

Comment:
You will also have to add the checks to the monolithic and server integration tests [here](https://github.com/abiquo/abiquo-cookbook/tree/master/test/integration), so we verify that the packages are actually installed, the services are running, and the ports are open.